### PR TITLE
Temporarily disable LOCAL scoped services

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

**Review by AleDore — COMMENTED** [review]
What about considering a flag that enables this kind of filter?

File: GetVisibleServices/handler.ts [hunk 1]
```diff
@@ -52,7 +53,7 @@ export function GetVisibleServicesHandler(
       maybeVisibleServicesJson => {
         const servicesTuples = toServicesTuple(
           new StrMap(maybeVisibleServicesJson.getOrElse({}))
-        );
+        ).filter(_ => _.scope === ServiceScopeEnum.NATIONAL); // Temporary disable Local services
```
Comment (pasqualedevita) [inline GetVisibleServices/handler.ts:56]: We can add an app setting with the list of enabled services by scope, for example
`FF_SERVICE_SCOPES="LOCAL,NATIONAL"`

Comment (BurnedMarshal) [reply GetVisibleServices/handler.ts]: Added a FF to enable NATIONAL scope filtering in 10f69cf. IMHO is useless a deep filtering configuration with FF.

Comment (pasqualedevita) [reply GetVisibleServices/handler.ts]: new app setting will be `FF_ONLY_NATIONAL_SERVICES` with true/false values?

Comment (BurnedMarshal) [reply GetVisibleServices/handler.ts]: Yes. The variable is optional if missing the filter is disabled.

Comment (pasqualedevita) [reply GetVisibleServices/handler.ts]: ok I will add this app setting on fn-apps and we can deploy only to one fn-app to test it

Comment (pasqualedevita) [reply GetVisibleServices/handler.ts]: FYI https://github.com/pagopa/io-infrastructure-live-new/pull/421

File: GetVisibleServices/__tests__/handler.test.ts [hunk 2]
```diff
@@ -85,7 +85,8 @@ describe("GetVisibleServicesHandler", () => {
       })
     };
     const getVisibleServicesHandler = GetVisibleServicesHandler(
-      blobStorageMock as any
+      blobStorageMock as any,
+      false
```
Comment (AleDore) [inline GetVisibleServices/__tests__/handler.test.ts:89]: Can we add a test when this param is `true`?

Comment (BurnedMarshal) [reply GetVisibleServices/__tests__/handler.test.ts:99]: Unit test added with ea72764

**Review by AleDore — APPROVED** [review]
lgtm

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**